src/lib/licence-transformer/nald-transformer.js

```diff
@@ -27,7 +27,7 @@ class NALDTransformer extends BaseTransformer {
 
     const mostRecentVersionPurposes = data.data.purposes.filter(purpose => purpose.AABV_INCR_NO !== data.data.versions.sort((a, b) => parseInt(b.INCR_NO) - parseInt(a.INCR_NO)))
 
-    const licenceHolderParty = find(currentVersion.parties, (party) => {
+    const licenceHolderParty = currentVersion.parties.find((party) => {
       return party.ID === currentVersion.ACON_APAR_ID
     })
```

Comment:
If we're okay with one-line syntax for `find` (as we've used elsewhere) then this and other similar ones could be one line:
```suggestion
    const licenceHolderParty = currentVersion.parties.find((party) => party.ID === currentVersion.ACON_APAR_ID)
```